.github/workflows/job_PreCommit_Python_Dataframes.yml

```diff
@@ -0,0 +1,33 @@
+name: Precommit Python Dataframe
+run-name: Python Precommit Dataframe
+on:
+  pull_request:
+    branches: [ "master", "release-*" ]
+    paths: [ "model/.*","sdks/python/.*","release/.*"]
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  PythonDataframe:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'Run Python_Dataframes PreCommit')
+    runs-on: self-hosted
+    defaults:
+      run:
+        working-directory: sdks/python
+
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.7 - 3.11'
```

Comment:
This will just install one of these versions, right? But we're trying to run 3.7, 3.8, 3.9, 3.10, and 3.11 tests; would we be better off doing a matrix or mapping this to multiple jobs?

Reply:
A similar question applies to other workflows